docs/defradb/guides/peer-to-peer.md

```diff
@@ -2,143 +2,275 @@
-Jan  2 10:15:49.163 INF node Providing HTTP API at http://127.0.0.1:9181 PlaygroundEnabled=false
-Jan  2 10:15:49.163 INF node Providing GraphQL endpoint at http://127.0.0.1:9181/api/v0/graphql
-```
+1. **Node Initialization with libp2p Host**
+    When starting a DefraDB node, it automatically sets up a libp2p host with a unique Peer ID derived from a secret private key:
 
-This host has a Peer ID, which is a function of a secret private key generated when the node is started for the first time. The Peer ID is important to know as it may be relevant for different parts of the peer-to-peer networking system. The libp2p networking stack can be enabled or disabled.
+    ```sh
+    $ defradb start
+    ...
+    Jan  2 10:15:49.124 INF cli Starting DefraDB
+    Jan  2 10:15:49.161 INF net Created LibP2P host PeerId=12D3KooWEFCQ1iGMobsmNTPXb758kJkFc7XieQyGKpsuMxeDktz4 Address=[/ip4/127.0.0.1/tcp/9171]
+    Jan  2 10:15:49.162 INF net Starting internal broadcaster for pubsub network
+    Jan  2 10:15:49.163 INF node Providing HTTP API at http://127.0.0.1:9181 PlaygroundEnabled=false
+    Jan  2 10:15:49.163 INF node Providing GraphQL endpoint at http://127.0.0.1:9181/api/v0/graphql
+    ```
 
-```bash
-$ defradb start --no-p2p
-```
+    - The Peer ID is critical for node discovery and participation in distributed document synchronization.
+    - The libp2p stack can be toggled on or off:
 
-The passive networking system can also be enabled or disabled. By default, if the P2P network is online, the passive networking system is turned on.
+    ```sh
+    defradb start --no-p2p
+    ```
 
-```bash
-$ defradb start --peers /ip4/0.0.0.0/tcp/9171/p2p/<peerID_of_node_to_replicate_to>
-```
+1. **Managing Passive Networking Behavior**
 
-A node automatically listens on multiple addresses or ports when the P2P module is instantiated. These are referred to as the peer-to-peer address, which is expressed as a multi-address. A multi-address is a string that represents a network address and includes information about the transport protocol and addresses for multiple layers of the network stack.
+   - To enable passive replication and specify target peers:
 
+   ```sh
+   defradb start --peers /ip4/0.0.0.0/tcp/9171/p2p/<peerID>
+   ```
 
-```bash
-/ip4/0.0.0.0/tcp/9171/p2p/<peerID_of_node_to_replicate_to>
+   - To change the peer-to-peer listening address:
 
-scheme/ip_address/protocol/port/protocol/peer_id
-```
-The peer listens in on the p2p port 9171​ by default, which can be customized through the CLI or the configuration file.
+   ```sh
+   defradb start --p2paddr /ip4/0.0.0.0/tcp/9172
+   ```
 
-```bash
-$ defradb start --p2paddr /ip4/0.0.0.0/tcp/9172
-```
+2. **Understanding Multi-Addresses**
+Multi-addresses define how a node communicates, detailing protocol and addressing layers:
+
+    ```sh
+    /ip4/0.0.0.0/tcp/9171/p2p/<peerID>
+    Format: scheme/ip_address/protocol/port/protocol/peer_id
+    ```
```

Comment:
todo: This section isn't rendering correctly, you can see it in the preview link (https://ashutosh-guides-peer-to-peer.docs-source-network.pages.dev/defradb/guides/peer-to-peer#passive-replication-features)

It might be because its indented